linux_os/guide/services/ssh/ssh_server/sshd_include_crypto_policy/tests/no_crypto.fail.sh

```diff
@@ -1,12 +1,7 @@
 #!/bin/bash
 
-# this is done because the remediation will reset the /etc/ssh/sshd_config file
-# which is modified by Automatus so that root can log in.
-# This prevents Automatus from logging in for final scan.
-echo "PermitRootLogin yes" > /etc/ssh/sshd_config.d/99-automatus.conf
-
-sed -i '/Include/d' /etc/ssh/sshd_config
+sed -i '/Include /etc/crypto-policies/back-ends/opensshserver.config/d' /etc/ssh/sshd_config /etc/ssh/sshd_config.d/*.conf
```

Comment:
You need to escape slashes or use a different sed separator than /.

Reply:
Fixed.